src/overlays/actors/ovl_En_Ru1/z_en_ru1.c

```diff
@@ -1271,21 +1271,21 @@ void func_80AED738(EnRu1* this, PlayState* play) {
 
 void func_80AED83C(EnRu1* this) {
     s32 pad[2];
-    Vec3s* tempPtr;
-    Vec3s* tempPtr2;
+    Vec3s* headRotPtr;
+    Vec3s* torsoRotPtr;
```

Comment:
```suggestion
    Vec3s* headRot;
    Vec3s* torsoRot;
```
Note this is how you name them in EnZl3 which are also pointers.